iogt/static/css/modules/_m-heading.scss

```diff
@@ -45,6 +45,7 @@
 	color: $victoria;
 	text-transform: uppercase;
 	font-size: $font-medium;
+	padding: 10px 15px;
```

Comment:
Are these changes meant for the smartphone designs, because you're changing them in the feature phone files?